src/monitors/analytics/messageReceived.ts

```diff
@@ -1,15 +0,0 @@
-import { ENABLE_INFLUX } from '#root/config';
```

Comment:
Where is this re- implemented?

Reply:
https://github.com/skyra-project/skyra/blob/7f39ff74b10bc15a784662741d13eb36948d5f09/src/lib/structures/events/AnalyticsEvent.ts#L12-L19